### ISSUE TITLE
Understanding Usage

### ISSUE THREAD

**Issue by jecourte** [eeoom/advanced_in_app_review#13]
Hi - Thanks for a great tool here.  I have a few questions around the parameters.   I'm using the simple example code in a FlutterFlow app inside my main() function:

AdvancedInAppReview()
      .setMinDaysBeforeRemind(20)
      .setMinDaysAfterInstall(5)
      .setMinLaunchTimes(5)
      .setMinSecondsBeforeShowDialog(4)
      .monitor();

When I call this code, when does the very first app review modal get called?  Does it have to match all of the "min" times before it gets launched for the first time, or does it launch when this code is called, and all subsequent presentations of the modal would adhere to the "min" times?  Also, for setMinSecondsBeforeShowDialog, is this just from the time you call the function?  Or is it "since the last call"?  

I ask because I called this code right away inside the main() function, thinking it would wait to adhere to all the parameters listed above before it showed, but in my Android app, it launched the model right away before the user even logged in.  

Thanks for any intel here.  Appreciate it.

**Comment by eeoom** [comment]
Yes, interesting. Normally all of the criteria have to match before the plugin tries to open the view.
